### PR TITLE
Proxy server

### PR DESCRIPTION
**Pull request by redperezz**
updated README

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+.DS_Stored
```
Comment (theofficetroll) [inline .gitignore:1]: Add linebreak.

File: README.md [hunk 2]
```diff
@@ -1 +1,48 @@
...
+
+## Development
+
+### Installing Dependencies
+
+From within the root directory:
+
+```sh
+npm install
+```
```
Comment (theofficetroll) [inline README.md:48]: Looks good.

File: teamfeedback/proxy-server/.babelrc [hunk 3]
```diff
@@ -0,0 +1,3 @@
+{
+  "presets": ["@babel/env", "@babel/preset-react"]
+}
```
Comment (theofficetroll) [inline teamfeedback/proxy-server/.babelrc:3]: Add linebreak.

File: teamfeedback/proxy-server/.gitignore [hunk 4]
```diff
@@ -0,0 +1,2 @@
+node_modules
+dist
```
Comment (theofficetroll) [inline teamfeedback/proxy-server/.gitignore:2]: Add linebreak.

File: teamfeedback/proxy-server/package.json [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+    "nodemon": "^2.0.4",
+    "style-loader": "^2.0.0",
+    "webpack": "^5.3.2",
+    "webpack-cli": "^4.1.0",
+    "webpack-dev-server": "^3.11.0"
+  },
+  "dependencies": {
+    "axios": "^0.20.0"
+  }
+}
```
Comment (theofficetroll) [inline teamfeedback/proxy-server/package.json:30]: Looks good. May want to update description and author.

File: teamfeedback/proxy-server/public/index.html [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+    <!-- <script src="http://localhost:3200/bundle.js"></script> Individual Items (primary) -->
+    <!-- <script src="http://localhost:3030"></script> Reviews (secondary) -->
+    <!-- <script src="http://localhost:3031/bundle.js"></script> Bonuses (primary) -->
+    <!-- <script src="http://localhost:3876"></script> Contributors (secondary) -->
+    <!-- <script src="http://localhost:3987" ></script> Charity (primary) -->
+    <!-- <script src="http://localhost:3201"></script> Purchase (secondary) -->
+    <script src="http://localhost:3201/dist/bundle/bundle.js"></script> System Requirements (secondary)
+  </body>
+  <footer></footer>
+</html>
```
Comment (theofficetroll) [inline teamfeedback/proxy-server/public/index.html]: Looks good.

File: teamfeedback/proxy-server/server.js [hunk 7]
```diff
@@ -0,0 +1,18 @@
...
+  res.sendFile(path.join(__dirname + '/public/index.html'), (err) => {
+    if (err) {
+      res.send(err);
+    }
+  });
+})
+
+app.listen(port, () => {
+  `App listening at http://localhost:${port}`
+});
```
Comment (theofficetroll) [inline teamfeedback/proxy-server/server.js:18]: Looks good.

**Review by theofficetroll — APPROVED** [review]
Looks good. Just some minor nits.

**Comment by theofficetroll** [comment]
Why are you merging submodules?
You should be able to just point at the bundle locations other than the API calls, which aren't currently working and will need defaulted out.
I'm probably missing something here, I'm curious what you're doing :-)

**Comment by redperezz** [comment]
Hey Mark!

Omg I didn't realize I didn't change it back! Thanks for catching that I
must have been messing with something but forgot to change it back lol.

**Comment by redperezz** [comment]
Oh wait that must be an old PR or maybe I didn't submit the current one but
yeah it's up to date now and will double check!

Thank you!

**Pull request merged**